datasets/librispeech_asr/README.md

```diff
@@ -63,7 +63,7 @@ LibriSpeech is a corpus of approximately 1000 hours of 16kHz read English speech
 
 ### Supported Tasks and Leaderboards
 
-- `automatic-speech-recognition`, `audio-speaker-identification`: The dataset can be used to train a model for Automatic Speech Recognition (ASR). The model is presented with an audio file and asked to transcribe the audio file to written text. The most common evaluation metric is the word error rate (WER). The task has an active leaderboard which can be found at https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-clean and ranks models based on their WER.
+- `automatic-speech-recognition`, `audio-speaker-identification`: The dataset can be used to train a model for Automatic Speech Recognition (ASR). The model is presented with an audio file and asked to transcribe the audio file to written text. The most common evaluation metric is the word error rate (WER). The task has an active Hugging Face leaderboard which can be found at https://huggingface.co/spaces/huggingface/hf-speech-bench and ranks models based on their WER.
```

Comment:
I would simply add the HF leaderboard and also keep the pwc link, wdyt ?

Also regarding Common Voice, I think we can do the same but let's see on slack since it's under the mozilla org

Reply:
Sure! Let's go with that :-) I'm not on the Slack channel for the Mozilla org, may I be added? Thanks!